lib/src/components/NxPagination/__tests__/NxPagination.test.tsx

```diff
@@ -85,105 +85,105 @@ describe('NxPagination', function() {
   describe('when there are five or fewer pages', function() {
     it('renders only arrows and buttons for the individual pages', function() {
       expect(getShallowComponent({ pageCount: 2, currentPage: 0 })).toMatchElement(
-        <div>
-          <NxButton>1</NxButton>
-          <NxButton>2</NxButton>
-          <NxButton><NxFontAwesomeIcon icon={{} as IconProp} /></NxButton>
-        </div>
+        <nav>
+          <NxButton aria-label="first page">1</NxButton>
+          <NxButton aria-label="page 2">2</NxButton>
+          <NxButton aria-label="next page"><NxFontAwesomeIcon icon={{} as IconProp} /></NxButton>
```

Comment:
`toMatchElement` does not check props.  You'll need to assert these separately.

If you're wondering about the `icon` prop on `NxFontAwesomeIcon`, it's just to satisfy the type checker.

Reply:
3cc14e7475ed7e7a139a659b51b5d4bb65cd5635

Reply:
> I guess I could have used a fragment instead. I just needed to have two components in here.

To me a fragment is always preferable to just a `<div>`, especially because some devs are likely to cut and paste the example code and we could end up with app code littered with unnecessary bare `<div>` tags.